crates/holochain_p2p/tests/tests/node_messaging.rs

```diff
@@ -298,26 +298,26 @@ async fn test_publish() {
     hc1.test_set_full_arcs(space.clone()).await;
     hc2.test_set_full_arcs(space.clone()).await;
 
+    let op = test_dht_op(holochain_types::prelude::Timestamp::now());
+    let op_hash = op.as_hash().clone();
+
+    // TODO invoking process_incoming_ops is a hack,
+    //      prefer calling a function on the mem store directly.
+    hc2.test_kitsune()
+        .space(dna_hash.to_k2_space())
+        .await
+        .unwrap()
+        .op_store()
+        .process_incoming_ops(vec![bytes::Bytes::from(
```

Comment:
The op only needs to be processed once. The loop just waits for publish to happen.